config/authorities/linked_data/loc.json

```diff
@@ -35,20 +33,20 @@
     "term_id": "ID",
     "language": ["en"],
     "results": {
-      "id_ldpath": "loc:lccn",
-      "label_ldpath": "skos:prefLabel :: xsd:string",
+      "id_ldpath":       "loc:lccn | madsrdf:code",
```

Comment:
This is the second change that allows the id to come from two different paths in the graph.